### PR TITLE
PRIME-1674 Admin view into license classes

### PR DESCRIPTION
**Pull request by AnaisHebert**

### CODE REVIEW

**Review by johnnyl6 — COMMENTED** [review]
seems college name column text wraps but plenty of whitespace left on the right side of the table

missing the back arrow returning to upper level page similar to email & toa maintenance.

image

File: prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<app-page>
```
Comment (hogmann) [inline prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html:1]: Added new functionality to app-page, you can use the following attributes and define a onBack to have the back arrow by the tittle.
[showToolbar]="true"
(back)="onBack()"

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html:1]: Also did read the thing above
haha

Comment (AnaisHebert) [reply prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html:1]: sweet

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html:1]: 🛥️

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.html:1]: I know talk real well

**Comment by Paahn** [comment]
> seems college name column text wraps but plenty of whitespace left on the right side of the table
> 
> missing the back arrow returning to upper level page similar to email & toa maintenance.
> 
> image

I tried removing the padding on the right, seems tight on smaller screens when there is a PractID filled in the last cell of the table

**Review by hogmann — DISMISSED** [review]
LGTM

**Review by Paahn — APPROVED** [review]
Looks great!
blake griffin 100

**Review by hogmann — APPROVED** [review]
LGTM

**Pull request merged**